From d0f8f60d0be26612e0773cd1427897ec183decdb Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Tue, 9 Oct 2018 14:25:10 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable CONFIG_F2FS_FS_ENCRYPTION.

Android Go device require this config for performance test,
enable it by default.

Test: ./performance_test/system/checkout_f2fs.sh

Change-Id: If6d07a82546759f672df8519d113a702c3bcfbe3
Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 8900d82c372e..a4add1c1f5dd 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -771,6 +771,7 @@ CONFIG_EXT4_FS_SECURITY=y
 CONFIG_F2FS_FS=y
 # CONFIG_F2FS_FS_POSIX_ACL is not set
 CONFIG_F2FS_FS_SECURITY=y
+CONFIG_F2FS_FS_ENCRYPTION=y
 CONFIG_QUOTA=y
 CONFIG_QUOTA_NETLINK_INTERFACE=y
 CONFIG_QFMT_V2=y
-- 
2.35.3

